### ISSUE TITLE
A pretty HTML template for the competition announcement email

### ISSUE THREAD

**Issue by AlphaSheep** [CubingZA/CubingZA-Website#141]
The current competition announcement emails are sent in a text format and are quite ugly. Let's revamp our competition announcement emails with a new and visually appealing HTML template. 

This update will not only make our announcements stand out but also provide a better user experience for our subscribers. Feel free to share any design ideas or suggestions to make our competition announcement emails more attractive and user-friendly.

**Comment by elfishhenry** [comment]
Starting to work on this.

**Comment by elfishhenry** [comment]
```html
![Uploading image_2024-08-12_201535749.png…]()
```
That should output something like this:

**Comment by AlphaSheep** [comment]
Yes, this is already much better! You could save this template to `server/components/notificationEmailer/notification.template.html`, and load it into `notificationEmailer.js` using Node's `fs.readFile` . You'd then need to do the necessary substitutions manually (using `.replace`) and set the `html` property of the object you send (see the Mailgun documentation [here](https://documentation.mailgun.com/docs/mailgun/api-reference/openapi-final/tag/Messages/#tag/Messages/operation/httpapi.(*apiHandler).handler-fm-18)).

If you do choose to do this, please work in a branch with this change only, because the email stuff needs extensive manual testing on a staging server that is hooked up to a Mailgun sandbox domain, and it will have to wait until I have some time to do that. I don't want that to hold up any other changes in the mean time.

**Comment by elfishhenry** [comment]
Alright, I'll see what I can do.

**Comment by elfishhenry** [comment]
I'll start this tomorrow.

**Comment by elfishhenry** [comment]
Should've fixed it.

**Comment by elfishhenry** [comment]
How should I continue?
